pkg/crypto/tinkcrypto/primitive/hybriddh/api/ecdhes_decrypt.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Is this file an exact copy from another project?
If yes, why can't it be imported?
If it's been changed, there needs to be a comment summarizing the change (and additional copyright headers).

I also feel like we should separate copied files into an internal structure like we did in fabric-sdk-go.